main/src/mill/modules/Jvm.scala

```diff
@@ -257,20 +261,52 @@ object Jvm {
     manifest.write(manifestOut)
     manifestOut.close()
 
-    def isSignatureFile(mapping: String): Boolean =
-      Set("sf", "rsa", "dsa").exists(ext => mapping.toLowerCase.endsWith(s".$ext"))
```

Comment:
Should we maintain this behavior in the new default `assemblyRules`? IIRC we added this because these files were causing problems for someone (there's an issue)

Reply:
Yep, I added it in default rules actually, right here: https://github.com/lihaoyi/mill/pull/309/files#diff-8987430aa3e4d7f9373fa4b3c3d5f2d0R13